support/routes/v2/isobot/ops/botService.ts

```diff
@@ -106,51 +71,11 @@ class BotService {
     }
   }
 
-  public verifySignature(
-    signature: string,
-    timestamp: string,
-    payload: SlackPayload
-  ) {
-    // Verify timestamp is within 5 minutes
-    const currentTime = new Date().getTime() / 1000
-    const slackTimestamp = parseInt(timestamp)
-
-    if (Math.abs(currentTime - slackTimestamp) > 300) {
-      return false
-    }
-
-    // Create HMAC with the signing secret and the timestamp
-    const signingSecret = config.get("slackbot.secret")
-    const versionNumber = "v0"
-    const reqBody = qs.stringify(payload, { format: "RFC1738" })
-    const signatureBasestr = `${versionNumber}:${timestamp}:${reqBody}`
-    const computedHash = createHmac("sha256", signingSecret)
-      .update(signatureBasestr)
-      .digest("hex")
-    const computedSig = `${versionNumber}=${computedHash}`
-
-    logger.info({
-      message: `Slack verification metadata`,
-      meta: {
-        currentTime,
-        slackTimestamp,
-        reqBody,
-      },
-    })
-
-    if (timingSafeEqual(Buffer.from(signature), Buffer.from(computedSig))) {
-      logger.info({ message: "Signature verified" })
-      return true
-    }
-    logger.error({ message: "Signature verification failed" })
-    return false
-  }
-
-  public async whitelistEmails(payload: SlackPayload) {
+  public async whitelistEmails(text: string) {
     // Sample user input:
     // email1,expDate email2,expDate
     // email1@xyz.com,2024-06-22 email2@abc.com,2025-01-31
```

Comment:
does this format make sense, feel free to suggest alternatives

Reply:
we can run with this first and let prod ops decide. tbh i think this won't quite work for >2-3 emails but idk how rare that would be